wwwroot/css/site.min.css

```diff
@@ -0,0 +1 @@
t-child{margin-bottom:0}@media(min-width:576px){.navbar-expand-sm .navbar-nav .nav-link{padding-left:4rem;padding-right:0}}html{font-size:14px}@media(min-width:768px){html{font-size:16px}}.border-top{border-top:1px solid #e5e5e5}.border-bottom{border-bottom:1px solid #e5e5e5}.box-shadow{box-shadow:0 .25rem .75rem rgba(0,0,0,.05)}button.accept-policy{font-size:1rem;line-height:inherit}html{position:relative;min-height:100%}body{margin-bottom:60px}.footer{position:absolute;bottom:0;width:100%;white-space:nowrap;line-height:60px}.post-list{max-width:800px;margin:auto;color:#333}.post-list a:link,.post-list a:visited{color:#333}.post-list a:hover{color:#338}.post-list-element{display:grid;grid-template-rows:auto;margin:70px 0 70px 0;grid-row-gap:5px;grid-column-gap:20px}.post-list-element .thumbnail{grid-area:thumbnail}.post-list-element .thumbnail img{max-height:100%;max-width:100%;outline:1px solid rgba(0,0,0,.2);outline-offset:-1px}.post-list-element .title{grid-area:title}.post-list-element .title h2{font-size:1.6rem;font-weight:500}.post-list-element .metadata{grid-area:metadata;text-align:right;font-weight:300}.post-list-element .metadata .author{font-size:1.2rem}.post-list-element .metadata .divider{font-size:1.2rem;color:#aaa}.post-list-element .metadata .date{font-size:1.2rem;color:#aaa}.post-list-element .tags{grid-area:tags}.post-list-element .tag{background-color:#400;color:#eee;display:inline-block;margin:4px 2px;padding:3px 10px;min-width:60px;border-radius:40px;font-size:.8rem;text-align:center}@media(min-width:0){.post-list-element{grid-template-columns:80px auto;grid-template-areas:"thumbnail title" "thumbnail tags" "metadata metadata"}.post-list-element .metadata .author{font-size:1.2rem;display:inline}.post-list-element .metadata .divider{font-size:1.2rem;display:inline}.post-list-element .metadata .date{font-size:1.2rem;display:inline}}@media(min-width:500px){.post-list-element{grid-template-columns:160px auto;grid-template-areas:"thumbnail title" "thumbnail tags" "metadata metadata"}.post-list-element .thumbnail{display:block}}@media(min-width:768px){.post-list-element{grid-template-columns:160px auto auto;grid-template-areas:"thumbnail title metadata" "thumbnail tags metadata"}.post-list-element .metadata .author{font-size:1.2rem;display:block}.post-list-element .metadata .divider{display:none}.post-list-element .metadata .date{font-size:1rem;display:block}}.post{max-width:800px;margin:auto;color:#333}.post .header .image{width:100%;object-fit:contain;outline:2px solid rgba(0,0,0,.2);outline-offset:-2px}.post .credit{font-size:1.2rem;color:#888;font-weight:300}.post .header{margin-bottom:3rem}.post .header .title{margin-bottom:.1rem}.post .header .credit{margin-bottom:2rem}.post .header a:link,.post .header a:visited{color:#888}.post .header a:hover{color:#88f}.post .body{font-family:Spectral,serif,"Segoe UI",sans-serif;font-size:1.2rem;line-height:2.2rem}.post .join{margin:4rem auto 4rem auto;text-align:center;font-size:1.6rem}
```

Comment:
These shouldn't be checked in (that's a maintenance nightmare!) but should be generated on-demand with whatever tool is doing it. (same with site.min.js)